### PR TITLE
ansible: end-to-end test

### PR DESCRIPTION
**Pull request by goneri**
- increase the amount of memory of the guest to 3000MB
- fix the create_domain, to it won't fail because host has an extra key
  that is ignored
- ssh_ping(): SSH key checking is irrelevant in our context

### CODE REVIEW

File: virt_lightning/shell.py [hunk 1]
```diff
@@ -117,10 +117,16 @@ def up(configuration, virt_lightning_yaml_path, context):
 def ansible_inventory(configuration, context):
     hv = vl.LibvirtHypervisor(configuration)
 
+    ssh_cmd_template = (
+        "{name} ansible_host={ipv4} ansible_username={username} "
+        'ansible_ssh_common_args="-o UserKnownHostsFile=/dev/null '
```
Comment (zloidemon) [inline virt_lightning/shell.py]: I see to redo this block with two ways:

1. Keep dictionary with options and convert to string
2. Add ssh configuration file and read file from ~/.config

File: virt_lightning/shell.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def up(configuration, virt_lightning_yaml_path, context):
         # that symbol more well add to check encoding block
         status_line += "🗲{name} ".format(**host)
         print(status_line)
-        domain = hv.create_domain(**host)
+        domain = hv.create_domain(name=host["name"], distro=host["distro"])
```
Comment (zloidemon) [inline virt_lightning/shell.py]: To avoid this issue you have to change create_domain signature to use `**kwargs` after defined named args.

Comment (goneri) [reply virt_lightning/shell.py]: I agree for the `**kwargs`. I preparing a better solution.

File: virt_lightning/shell.py (hunk 1 above)
Comment (goneri) [reply virt_lightning/shell.py]: I assume you mean ~/.ssh/config, I would prefer to keep this directory intact.

File: virt_lightning/shell.py (hunk 2 above)
Comment (goneri) [reply virt_lightning/shell.py]: update:

I don't thing we need `**kwargs` for this specific case and the current patch fix a problem. So I would prefer we apply the current patch. In a second time, we will prepare a better fix. WDYT?

**Comment by goneri** [comment]
I just did a end-to-end run in 5.1m :-)

**Pull request merged**